sign/dilithium/mode1/internal/params_test.go

```diff
@@ -0,0 +1,86 @@
+package internal
+
+import (
+	"testing"
+
+	"github.com/cloudflare/circl/internal/shake"
+	common "github.com/cloudflare/circl/sign/dilithium/internal"
+)
+
+// Tests specific to the current mode
+
+func hash(in []byte) [16]byte {
+	var ret [16]byte
+	h := shake.NewShake256()
+	h.Write(in)    //nolint:errcheck
+	h.Read(ret[:]) //nolint:errcheck
+	return ret
+}
+
+func TestNewKeyFromSeed(t *testing.T) {
```

Comment:
You can unify this test as it was done in the KAT_test file.

Reply:
Done.